jscomp/syntax/ast_external_attributes.ml

```diff
@@ -650,7 +650,7 @@ let handle_attributes
 
       | {set_index = true; _}
         ->
-        Bs_syntaxerr.err loc Conflict_ffi_attribute
+        Bs_syntaxerr.err loc (Conflict_ffi_attribute (Some "Attribute found that conflicts with [@@set_index]"))
```

Comment:
this is great `[@@bs.set_index]`, thanks!
think twice, lets keep changeset minimal, just have `Conflict_ffi_attribute "blabla"` should be fine